lib/ibge.rb

```diff
@@ -6,6 +6,7 @@ module Ibge
   DATA_PATH = File.expand_path(File.join(__dir__, '..', 'data'))
 
   module Reader
+    autoload :Helpers,      'ibge/reader/helpers'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.